src/layers/domain/repository/cpm_product_repository/tests/test_product_repository.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+from domain.core.cpm_product import CpmProduct
+from domain.core.cpm_system_id.v1 import ProductId
+from domain.core.root.v3 import Root
+from domain.repository.cpm_product_repository.v3 import CpmProductRepository
+from domain.repository.keys.v3 import TableKey
+from domain.repository.marshall import marshall_value
+from event.aws.client import dynamodb_client
+
+from test_helpers.terraform import read_terraform_output
+from test_helpers.uuid import consistent_uuid
+
+
+def _create_product_team(
+    seed: int = 1, name: str = "product-team-name", ods_code: str = "ABC"
+):
+    org = Root.create_ods_organisation(ods_code=ods_code)
+    return org.create_product_team(id=consistent_uuid(seed), name=name)
+
+
+@pytest.mark.integration
+def test__query_products_by_team():
```

Comment:
Maybe name test__query_products_by_product_team to be consistent with language, and makes it super easy to find the test for that function